Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -663,8 +663,15 @@ class BrowserViewController: UIViewController {
         urlBar.currentURL = url
         urlBar.leaveOverlayMode()
 
-        if let tab = tabManager.selectedTab,
-           let nav = tab.loadRequest(NSURLRequest(URL: url)) {
+        guard let tab = tabManager.selectedTab else {
+            return
+        }
+
+        if #available(iOS 9.0, *) {
+            tab.webView?.customUserAgent = nil
```

Comment:
This will reset the spoofed user agent whenever a new URL is entered manually or a link is selected from the home panel.